game.py

```diff
@@ -229,21 +265,21 @@ def handle_collision(self):
     def create_object_list(self):
         if self.tick_counter == 10 or self.tick_counter == 35:
             self.soon_to_draw = Element(self)
```

Comment:
As this is only used in this function, there's no need to make it an instance variable. 

A better change would've been to rename `self.soon_to_draw` to fit the other uses.

Reply:
Done, replace it with `pending_element`